src/studySetup.js

```diff
@@ -37,7 +37,7 @@ const baseStudySetup = {
   // telemetry
   telemetry: {
     // default false. Actually send pings.
-    send: false,
+    send: true,
```

Comment:
I'd rather keep this false, so we don't confuse the shield folks, since I'd been confusing them earlier, but I see how we should test the actual pings, so it's fine.

Reply:
Was this confusing to Shield folks because we were using the old extension ID? Otherwise I wonder what they would find confusing, given that we set the testing flag to true. That sounds like an issue on their end :)

Reply:
Yeah, it was an issue on their end - I believe they weren't quite prepared to filter out the testing pings